src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter((el) => century
+    ? el.sex === 'm' && Math.ceil(el.died / 100) === century : el.sex === 'm');
+
+  const len = men.length;
+  const ages = men.map(a => a.died - a.born);
+
+  const result = ages.reduce((a, b) => a + b) / len;
+
+  return result;
```

Comment:
You can return value immediately 
```suggestion
  return ages.reduce((a, b) => a + b) / len;;
```